packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -322,15 +318,25 @@ type RestrictedMenuItemProps = {
   dataFences?: TDataFence[];
   children: ReactNode;
 };
+type TLongLivedFlag = {
+  value: boolean;
+  reason?: string;
+};
 const restrictedMenuItemDefaultProps: Pick<
   RestrictedMenuItemProps,
   'permissions'
 > = {
   permissions: [],
 };
+function isLongLivedFlag(
```

Comment:
As flopflip can't know what a service such as LaunchDarkly returns and as a result is exposed to an application it feels safest to type guard it instead of using generics only.